examples/src/main/python/ml/generalized_linear_regression_example.py

```diff
@@ -17,9 +17,10 @@
 
 from __future__ import print_function
 
-from pyspark.sql import SparkSession
 # $example on$
 from pyspark.ml.regression import GeneralizedLinearRegression
+from pyspark.sql import SparkSession
+
```

Comment:
It's weird to have one blank line inside of the example and one outside, generally we seem to be leaving the blank lines out of the others.